crates/hyperqueue/src/dashboard/ui/screens/home/worker_info_table.rs

```diff
@@ -0,0 +1,110 @@
+use crate::dashboard::data::DashboardData;
+use crate::dashboard::ui::terminal::DashboardFrame;
+use crate::dashboard::ui::widgets::table::{StatefulTable, TableColumnHeaders};
+use crate::WorkerId;
+use tako::messages::common::WorkerConfiguration;
+use tui::layout::{Constraint, Rect};
+use tui::widgets::{Cell, Row};
+
+#[derive(Default)]
+pub struct WorkerInfoTable {
+    table: StatefulTable<WorkerInfoDataRow>,
+    current_worker_id: Option<WorkerId>,
+}
+
+#[derive(Default, Debug)]
+struct WorkerInfoDataRow {
+    pub label: String,
```

Comment:
```suggestion
    pub label: &'static str,
```
No need for this to be a heap-allocated `String`, since the labels are constant strings.